src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+    <script src="./index.js" defer></script>
+</head>
+
+<body>
+    <div id="toDoList">
+        <div class="header">Today's To Do</div>
+        <div class="addContainer">
+            <input type="text" class="taskAdd" id="myInput" placeholder="Add to your list...">
+            <button type="button" id="btnAdd">Add</button>
+        </div>
+        <ul id="taskList" class="toDoTasks"></ul>
+        <div class="footer">Clear all completed</div>
+    </div>
```

Comment:
I think is better to use semantics tags to me your code more readable, instead of  `div ` you can use `main` tags for the container on line 13 and a section for the children element of your `div` tag with the id of toDoList.

Reply:
Good advice, it was implemmented.